fixtures/hive/bee-50-c.json

```
@@ -0,0 +1,547 @@
+          "comment": "1/8 +Green Daylight"
+        },
+        {
+          "dmxRange": [51, 60],
+          "type": "Generic",
+          "comment": "Full +Green Tungsten"
+        },
+        {
+          "dmxRange": [61, 70],
+          "type": "Generic",
+          "comment": "1/2 +Green Tungsten"
+        },
+        {
+          "dmxRange": [71, 80],
+          "type": "Generic",
+          "comment": "1/4 +Green Tungsten"
+        },
+        {
+          "dmxRange": [81, 90],
+          "type": "Generic",
+          "comment": "1/8 +Green Tungsten"
+        },
+        {
+          "dmxRange": [91, 100],
+          "type": "Generic",
+          "comment": "Full -Green Daylight"
+        },
+        {
+          "dmxRange": [101, 110],
+          "type": "Generic",
+          "comment": "1/2 -Green Daylight"
+        },
+        {
+          "dmxRange": [111, 120],
+          "type": "Generic",
+          "comment": "1/4 -Green Daylight"
+        },
+        {
+          "dmxRange": [121, 130],
+          "type": "Generic",
+          "comment": "1/8 -Green Daylight"
+        },
+        {
+          "dmxRange": [131, 140],
+          "type": "Generic",
+          "comment": "Full -Green Tungsten"
+        },
+        {
+          "dmxRange": [141, 150],
+          "type": "Generic",
+          "comment": "1/2 -Green Tungsten"
+        },
+        {
+          "dmxRange": [151, 160],
+          "type": "Generic",
+          "comment": "1/4 -Green Tungsten"
+        },
+        {
+          "dmxRange": [161, 170],
+          "type": "Generic",
+          "comment": "1/8 -Green Tungsten"
+        },
+        {
+          "dmxRange": [171, 180],
+          "type": "Generic",
+          "comment": "Full CTS Daylight"
+        },
+        {
+          "dmxRange": [181, 190],
+          "type": "Generic",
+          "comment": "1/2 CTS Daylight"
+        },
+        {
+          "dmxRange": [191, 200],
+          "type": "Generic",
+          "comment": "1/4 CTS Daylight"
+        },
+        {
+          "dmxRange": [201, 210],
+          "type": "Generic",
+          "comment": "1/8 CTS Daylight"
+        },
+        {
+          "dmxRange": [211, 220],
+          "type": "Generic",
+          "comment": "Full CTS Tungsten"
+        },
+        {
+          "dmxRange": [221, 230],
+          "type": "Generic",
+          "comment": "1/2 CTS Tungsten"
+        },
+        {
+          "dmxRange": [231, 240],
+          "type": "Generic",
+          "comment": "1/4 CTS Tungsten"
+        },
+        {
+          "dmxRange": [241, 255],
+          "type": "Generic",
+          "comment": "1/8 CTS Tungsten"
+        }
+      ]
+    },
+    "EFX": {
+      "name": "Effect",
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "dmxRange": [0, 10],
+          "type": "Generic",
+          "comment": "OFF"
+        },
+        {
+          "dmxRange": [11, 28],
+          "type": "Generic",
+          "comment": "Police Car 1"
```

```suggestion
          "type": "Effect",
          "effectName": "Police Car 1"
```

For the following capabilities as well, please.